go/engine/common_test.go

```diff
@@ -144,13 +144,14 @@ func CreateAndSignupFakeUserPaper(tc libkb.TestContext, prefix string) *FakeUser
 	return fu
 }
 
-func CreateAndSignupFakeUserSafe(g *libkb.GlobalContext, prefix string) (*FakeUser, error) {
+func CreateAndSignupFakeUserSafe(g *libkb.GlobalContext, prefix string, fmod func(*SignupEngineRunArg)) (*FakeUser, error) {
```

Comment:
Make this simpler -- split into `CreateAndSignupFakeUserSafe` and `CreateAndSignupFakeUserSafeWithArg`, and have the former call the later. And your test calls the latter.

Reply:
i like it. done!